assets/javascripts/src/components/pages/Contribution.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 export default class Contribution extends React.Component {
     constructor(props) {
-        super(props)
```

Comment:
To be honest I'm not sure. The js build pipeline is still dark magic to me :smile: